sonar-db/src/main/java/org/sonar/db/qualitygate/ProjectQgateAssociationDao.java

```diff
@@ -46,8 +47,7 @@ public ProjectQgateAssociationDao(MyBatis mybatis) {
     }
   }
 
-  @VisibleForTesting
-  List<ProjectQgateAssociationDto> selectProjects(ProjectQgateAssociationQuery query, Long gateId) {
+  public List<ProjectQgateAssociationDto> selectProjects(ProjectQgateAssociationQuery query, Long gateId) {
     return selectProjects(query, gateId, 0, Integer.MAX_VALUE);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "selectProjects"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)